README.md

```diff
@@ -171,6 +171,14 @@ $.Error :: Type
 Type comprising every Error value, including values of more specific
 constructors such as [`SyntaxError`][2] and [`TypeError`][3].
 
+#### `FiniteNumber`
+
+Type comprising every Number value except `Infinity`, `-Infinity` and `NaN`.
```

Comment:
Shall we ``s/Number/[`Number`](#number)/`` to make the link between the types absolutely clear?